src/common/load-plugins.js

```diff
@@ -100,7 +100,7 @@ function findPluginsInNodeModules(nodeModulesDir) {
       "@*/prettier-plugin-*/package.json",
       "@prettier/plugin-*/package.json"
     ],
-    { cwd: nodeModulesDir }
+    { cwd: nodeModulesDir, expandDirectories: false }
```

Comment:
Is `expandDirectories: false` really needed here?

Reply:
yes